[Docs]Update instructions.md (#2508)
exercism/javascript: exercises/concept/coordinate-transformation/.docs/instructions.md

# Instructions

> Your design company has primarily been working with CSS transformations to build web pages. After some discussion, a
> decision is made to start using JavaScript to perform some calculations dynamically. Some of your teammates are less experienced with
> JavaScript, so you decide to use a function closure to create reusable transformation for `{x, y}` coordinate pairs.

Reviewer: I meant this: https://exercism.org/docs/building/markdown/markdown

```suggestion
Your design company has primarily been working with CSS transformations to build web pages.
After some discussion, a decision is made to start using JavaScript to perform some calculations dynamically.
Some of your teammates are less experienced with JavaScript, so you decide to use a function closure to create reusable transformation for `{x, y}` coordinate pairs.
```

etc ;)

Author: I see.
I feel like I managed this time around.

Implement the `translate2d` function that returns a function making use of a closure to perform a repeatable 2d translation of a coordinate pair.

> > In Geometry, [translation][wiki-translate] reffers to moving points, vectors or shapes the same distance in one
> > direction. It can be interpreted as addition of a constant to every point.

Reviewer: ```suggestion
> In Geometry, [translation][wiki-translate] refers to moving points, vectors or shapes the same distance in one direction.
> It can be interpreted as addition of a constant to every point.
```

Implement the `scale2d` function that returns a function making use of a closure to perform a repeatable 2d scale of a coordinate pair.

> > In geometry, uniform [scaling][wiki-scale] reffers to enlarging or shrinking vectors or shapes in the same
> > diraction. It can be interpreted as multiplying every point by a constant (scaling factor).
> 

Reviewer: ```suggestion
> In geometry, uniform [scaling][wiki-scale] refers to enlarging or shrinking vectors or shapes in the same direction. 
> It can be interpreted as multiplying every point by a constant (scaling factor).
> 
```

> > 

Author: Ahh, now it breaks the styling check. 
I think it doesn't like the empty line between the two quotes

Reviewer: That's okay. Let's see if we can auto fix it.

Author: magic spells i see.
Good
